common/tools/dev-tool/src/commands/run/extract-api.ts

```diff
@@ -28,7 +28,7 @@ export const commandInfo = makeCommandInfo(
     "subpath-doc-model": {
       shortName: "sdc",
       kind: "boolean",
-      default: false,
+      default: true,
```

Comment:
should we remove this option and always generate for subpaths now?

Reply:
Have not reviewed the PR yet but if CI is happy and can read the new format (looks like a large concatenated json? Is that right?) then definitely! It's only here as a stop-gap so we can generate api.md files for subpath exports without breaking CI (which expects a single json file)

Reply:
I see, yeah then we need some fix if CI expects a single api.json

Reply:
For now I worked around it by removing sub path api.json files and renaming the merged file.  Hopefully we could just simply flip to the merged file at some point later.

Reply:
> can read the new format (looks like a large concatenated json? Is that right?)

For now we just generate the json with the merged entrypoints but not using it yet. The original `.api.json` is still there and still the only one with the extension.